doc/netplan-tutorial.md

```diff
@@ -0,0 +1,560 @@
+3: enp6s0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN mode DEFAULT group default qlen 1000
+    link/ether 00:16:3e:7b:68:9d brd ff:ff:ff:ff:ff:ff
+```
+
+In this case, `enp5s0` is the primary interface connected to the default LXD network and `enp6s0` is the second interface you added connected to your custom network.
+
+
+Now, let's start with a simple exercise.
+
+## Running netplan for the first time
+
+Start by typing the command `netplan` in your shell:
+
+```bash
+netplan
+```
+
+You will see the output below
+
+```
+You need to specify a command
+usage: /usr/sbin/netplan  [-h] [--debug]  ...
+
+Network configuration in YAML
+
+options:
+  -h, --help  show this help message and exit
+  --debug     Enable debug messages
+
+Available commands:
+
+    help      Show this help message
+    apply     Apply current netplan config to running system
+    generate  Generate backend specific configuration files from /etc/netplan/*.yaml
+    get       Get a setting by specifying a nested key like "ethernets.eth0.addresses", or "all"
+    info      Show available features
+    ip        Retrieve IP information from the system
+    set       Add new setting by specifying a dotted key=value pair like ethernets.eth0.dhcp4=true
+    rebind    Rebind SR-IOV virtual functions of given physical functions to their driver
+    try       Try to apply a new netplan config to running system, with automatic rollback
+```
+
+As you can see, netplan has a number of sub commands. Let's explore some of them.
+
+## Showing your current netplan configuration
+
+To show your current configuration, run the command `netplan get`.
+
+```bash
+netplan get
+```
+
+You should see an output similar to the one below:
+
+
+```yaml
+network:
+  version: 2
+  ethernets:
+    enp5s0:
+      dhcp4: true
+```
+
+It shows you have an ethernet interface called `enp5s0` and it has DHCP enabled for IPv4.
+
+## Checking the file where your configuration is stored
+
+The configuration you just listed is stored at `/etc/netplan`. You can see the contents of the file with the command below:
+
+```bash
+cat /etc/netplan/50-cloud-init.yaml
+```
+
+You should see an output similar to this:
+
+```yaml
+# This file is generated from information provided by the datasource.  Changes
+# to it will not persist across an instance reboot.  To disable cloud-init's
+# network configuration capabilities, write a file
+# /etc/cloud/cloud.cfg.d/99-disable-network-config.cfg with the following:
+# network: {config: disabled}
+network:
+    version: 2
+    ethernets:
+        enp5s0:
+            dhcp4: true
+```
+
+This file was automatically generated by `cloud-init` when the system was initialized. As noted in the comments, changes to this file will not persist.
+
+## Enabling your second network interface with DHCP
+
+There are basically 2 ways to create or change netplan configuration: 1) using the `netplan set` command and 2) editing the YAML files manually.
```

Comment:
nitpick: "1)" and "2)" should be rendered as an actuall ordered list in markdown.